tests/certs/mtls/client/Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: all clean
+
+client.key:
+	openssl genrsa -out $@ 2048
+
+client.csr: client.key
+	openssl req -key $< -new -out $@ -config cert.cnf
+
+client.pem: client.csr
+	openssl x509 -req -CA ./ca/ca.crt -CAkey ./ca/ca-private.key -in client.csr -outform PEM -out client.pem -days 730 -CAcreateserial
```

Comment:
nit; any reason we're using 730 days here v 7300 days above in [tests/certs/expired/ca/Makefile](https://github.com/psf/requests/pull/6662/files#diff-44c61b9e0b9f24c770f652b5da6da9ce7c402c4aa1b6dbc47324fbe86cead1c8)?

Not an issue but we may want to keep them consistent or note the variance as significant.

Reply:
Certificate authorities should last significantly longer (20-25 years) per CAB forum. This is a cert issued by that which could theoretically live longer than the CA if we did the same expiration. I figured ~2 years would be sufficient for this test while helping us make sure the make targets keep working